_includes/component_grid_core.html

```diff
@@ -1,4 +1,4 @@
-<div class="grid-core" style="background-image: url(img/{{ include.data.id }}.png);">
+<div class="grid-core" style="background-image: url(../img/{{ include.data.id }}.png);">
```

Comment:
You are right. this does bring good question. we should decide on one way doing the path and be consistance with those.